test/cpp/qps/usage_timer.cc

```diff
@@ -48,6 +51,23 @@ static double time_double(struct timeval* tv) {
   return tv->tv_sec + 1e-6 * tv->tv_usec;
 }
 
+static void get_cpu_usage(unsigned long long* total_cpu_time,
+                          unsigned long long* idle_cpu_time) {
+  std::ifstream proc_stat("/proc/stat");
```

Comment:
This is Linux-specific, right? Can you wrap this in the appropriate #ifdef dance so that other platforms don't complain?